### ISSUE TITLE
Replay page: Collection Deleting all sessions is super slow

### ISSUE THREAD

**Issue by bebiksior** [caido/caido#1321]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Bug description
Today, I was trying to remove a collection with lots of Replay Sessions (around 1000) and it was super slow. I could see every session being slowly deleted, here's video:

https://github.com/user-attachments/assets/7e2e37ad-4465-4619-b251-b308504e0218

Also, during this process my Caido instance was completely unusable. I guess it could be a UI issue, after restarting Caido every session was gone.

### How to reproduce
1. Create a large collection with lots of sessions on the Replay Page
2. Hover over your Collecion
3. click 3 dots -> Delete Sessions (x)
4. Notice that sessions are being deleted very slowly

### Environment & setup

OS: 
OS version: 
Caido client: 
Caido version:

**Comment by bebiksior** [comment]
Also, I wanted to clear my default collection because sending requests to replay page was starting to slow down (it took ~1.5s). I think some optimizations could be done there. I guess that creating a new collection and sending requests there might fix the issue, but I haven’t tried it

**Comment by Sytten** [comment]
Yeah we patched that by doing a loop I am pretty sure because we didn't have a mass delete API.
